### ISSUE TITLE
arm64: missing register in regs_access() for ldr instruction

### ISSUE THREAD

**Issue by find0x90** [aquynh/capstone#2137]
The regs_access() function is missing the read register for the `ldr` instruction. 

Below is a small script that reproduces the issue between version 4.0.2 and the most recent commit as of this comment. 

```python
#! /usr/bin/env python3
# cs_test.py

from capstone import *

md = Cs(CS_ARCH_ARM64, CS_MODE_ARM | CS_MODE_LITTLE_ENDIAN)
md.detail = True

instruction_bytes = b"\x00\x05\x45\xf9"

inst = list(md.disasm(instruction_bytes, offset=0x0, count=1))[0]

print(inst)

regs_read, regs_written = inst.regs_access()
regs_read = [inst.reg_name(r) for r in regs_read]
regs_written = [inst.reg_name(r) for r in regs_written]

print(regs_read, regs_written)
```

4.0.2:
```
$ ./cs_test.py
<CsInsn 0x0 [000545f9]: ldr x0, [x8, #0xa08]>
['x8'] ['x0']
```

next branch: 36c5eef4ed8457e454fc310fac3b5f836dd15aa7
```
> ./cs_test.py
<CsInsn 0x0 [000545f9]: ldr x0, [x8, #0xa08]>
[] ['x0']
```

**Comment by XVilka** [comment]
Should be addressed by https://github.com/capstone-engine/capstone/pull/2026 cc @Rot127

**Comment by Rot127** [comment]
> Should be addressed by https://github.com/capstone-engine/capstone/pull/2026 cc @Rot127

It is (ignore that the memory operand is not marked as such. This will be fixed).

```
./cstool -d aarch64 \x00\x05\x45\xf9
 0  00 05 45 f9  ldr	x0, [x8, #0xa08]
	ID: 581 (ld3)
	op_count: 2
		operands[0].type: REG = x0
		operands[0].access: WRITE
			Vector Arrangement Specifier: 0x0
			Vector Index: 0
		operands[1].type: REG = x8
		operands[1].access: READ
			Vector Arrangement Specifier: 0x0
			Vector Index: 0
	Registers read: x8
	Registers modified: x0

```

**Comment by find0x90** [comment]
Fixed for me on the next branch: e3a2b4c4. Thanks! I'll close this out.
